Result+Povio.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Result+Povio.swift
+//  Alamofire
+//
+//  Created by Borut Tomažin on 24/02/2020.
+//  Copyright © 2020 Povio Labs. All rights reserved.
+//
+
+import Foundation
+
+extension Result where Success == Void {
+  static func success() -> Result {
```

Comment:
Maybe computed prop instead?

Reply:
I would stick with the function instead. Since we have a function for non-void cases and also for failures, it would only bring confusion.